src/structures/MessageAttachment.js

```diff
@@ -43,6 +43,21 @@ class MessageAttachment {
     return this;
   }
 
+  /**
+   * Sets whether this attachment is a spoiler
+   * @param {boolean} spoiler Whether the attachment should be marked as a spoiler
+   * @returns {MessageAttachment} This attachment
+   */
+  setSpoiler(spoiler) {
+    if (spoiler === true && this.spoiler === false) {
+      this.name = `SPOILER_${this.name}`;
+      return this;
+    } else {
+      if (this.spoiler === true) this.name.replace('SPOILER_', '');
+      return this;
+    }
```

Comment:
```suggestion
    }
    if (this.spoiler === true) this.name.replace('SPOILER_', '');
    return this;
```

That's not quite what I meant 😆 The else is redundant since the first  if block always returns